test/infrastructure/docker/docker/machine.go

```diff
@@ -438,17 +457,34 @@ func (m *Machine) machineImage(version *string) string {
 func logContainerDebugInfo(ctx context.Context, name string) {
 	log := ctrl.LoggerFrom(ctx)
 
-	cmd := exec.CommandContext(ctx, "docker", "inspect", name)
-	output, err := exec.CombinedOutputLines(cmd)
+	cli, err := types.GetDockerClient()
+	if err != nil {
+		log.Info("Error getting docker client", "name", name, "error", err)
+		return
+	}
+
+	containerInfo, err := cli.ContainerInspect(ctx, name)
+	if err != nil {
+		log.Info("Error inspecting container", "name", name, "error", err)
+		return
+	}
+	log.Info("Inspected the machine container", "output", fmt.Sprintf("%+v", containerInfo))
+
+	options := dockertypes.ContainerLogsOptions{
+		ShowStdout: true,
+		ShowStderr: true,
+	}
+	responseBody, err := cli.ContainerLogs(ctx, containerInfo.ID, options)
 	if err != nil {
-		log.Error(err, "Failed inspecting the machine container", "output", output)
+		log.Info("Error getting container logs", "name", name, "error", err)
+		return
 	}
-	log.Info("Inspected the machine container", "output", output)
+	defer responseBody.Close()
 
-	cmd = exec.CommandContext(ctx, "docker", "logs", name)
-	output, err = exec.CombinedOutputLines(cmd)
+	logs, err := io.ReadAll(responseBody)
 	if err != nil {
-		log.Error(err, "Failed to get logs from the machine container", "output", output)
+		log.Info("Error reading container logs", "name", name, "error", err)
+		return
 	}
-	log.Info("Got logs from the machine container", "output", output)
+	log.Info("Got logs from the machine container", "output", fmt.Sprintf("%+v", logs))
```

Comment:
This is seems to return a blob of bytes which isn't super useful

Reply:
Can't remember now why I would have chosen `%+v`, but that's obviously not right. Maybe copy/paste error. Will update.